cityofaustin/atd-moped
# Search Signal Projects from Signal Dashboard View

## Associated issues

https://github.com/cityofaustin/atd-data-tech/issues/7966

## Testing
**URL to test:** 
https://7966-search-signal-projects--atd-moped-main.netlify.app/moped/dashboard

**Steps to test:**
Material table has a search option in the upper right hand corner of table. Use it to see what you can find in the table. 

---
#### Ship list
